<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>783</width>
                <height>540</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="Eb3ImageQLabel" name="movieImgLabel">
            <property name="geometry">
                <rect>
                    <x>270</x>
                    <y>0</y>
                    <width>510</width>
                    <height>512</height>
                </rect>
            </property>
            <property name="text">
                <string>TextLabel</string>
            </property>
            <property name="scaledContents">
                <bool>true</bool>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QTreeView" name="experimentsTreeView">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>151</width>
                    <height>511</height>
                </rect>
            </property>
            <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
        </widget>
        <widget class="QPushButton" name="exportPandasButton">
            <property name="geometry">
                <rect>
                    <x>150</x>
                    <y>471</y>
                    <width>121</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Export Pandas</string>
            </property>
        </widget>
        <widget class="QLabel" name="currImageLabel">
            <property name="geometry">
                <rect>
                    <x>160</x>
                    <y>520</y>
                    <width>581</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
    </widget>
    <customwidgets>
        <customwidget>
            <class>Eb3ImageQLabel</class>
            <extends>QLabel</extends>
            <header>gui_eb3_image_qlabel</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
